### PR TITLE
Revamp for Mule 4

### PR DESCRIPTION
**Pull request by hannanelson**
[20-after-Use Object Store v2 _ MuleSoft Documentation.pdf](https://github.com/mulesoft/docs-object-store/files/4900211/20-after-Use.Object.Store.v2._.MuleSoft.Documentation.pdf)

### CODE REVIEW

**Review by prafulrana — DISMISSED** [review]

File: modules/ROOT/pages/osv2-guide.adoc [hunk 1]
```diff
@@ -3,141 +3,120 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Object Store v2 (Object Store v2) lets you share data and states across Mule runtime workers within a single application.
+Anypoint Object Store v2 (Object Store v2) enables you to share data and states across Mule runtime workers within a single application.
 
-Object Store v2 is enabled by default in the Anypoint Platform EU Control Plane and in Mule 4.
-
-In Mule 3, enable Object Store v2 from *Anypoint Platform > Runtime Manager* for each application you want to access Object Store v2.
+Object Store v2 is enabled by default in Mule 4 and in the Anypoint Platform EU Control Plane.
```
Comment (kbostrom) [inline modules/ROOT/pages/osv2-guide.adoc]:
```suggestion
Object Store v2 is enabled by default in Mule 4 and in Anypoint Platform EU Control Plane.
```

File: modules/ROOT/pages/osv2-guide.adoc [hunk 2]
```diff
@@ -3,141 +3,120 @@ ifndef::env-site,env-github[]
...
-Exchange from within Anypoint Studio. After installed, search Studio
-for `objectstore`. The Object Store connector works with both
-Object Store version 1 and version 2. Use Studio to create an application
-that you deploy to Runtime Manager where the application runs and serves
-requests to read to and write from the object store.
+Object Store Connector works with both Object Store version 1 and version 2.
 +
-The Object Store Reference field for Object Store Connector (`objectStore-ref` element in XML) must
-be set to `_defaultUserObjectStore` for data to appear in Object Store v2.
+To use Object Store v2 with Mule 4 apps, do not select an object store global element from the Object store menu.
```
Comment (kbostrom) [inline modules/ROOT/pages/osv2-guide.adoc]:
```suggestion
To use Object Store v2 with Mule 4 apps, do not select an object store global element from the *Object store* menu.
```

File: modules/ROOT/pages/osv2-guide.adoc [hunk 3]
```diff
@@ -3,141 +3,120 @@ ifndef::env-site,env-github[]
...
+When you enable Object Store v2, the *Runtime Manager > Application Data* option changes to *Object Store*.
+From the *Object Store* page, you can manage objects and their values in the object store.
 
-=== About Protecting Object Store Data
+=== Protect Object Store Data
 
-Object Store v2 uses TLS for secure transport. Data at rest is stored using FIPS 140-2 compliant encryption standards. If you require a higher level of security, we recommend that you encrypt sensitive data before writing it to the Object Store.
+Object Store v2 uses TLS for secure transport.
+Data at rest is stored using FIPS 140-2-compliant encryption standards.
+If you require a higher level of security, encrypt sensitive data before writing it to the Object Store.
```
Comment (kbostrom) [inline modules/ROOT/pages/osv2-guide.adoc]:
```suggestion
If you require a higher level of security, encrypt sensitive data before writing it to Object Store.
```

File: modules/ROOT/pages/osv2-guide.adoc [hunk 4]
```diff
@@ -3,141 +3,120 @@ ifndef::env-site,env-github[]
...
 
-. In Anypoint Studio, click the Exchange icon in the Studio task bar.
-. Click Login in Anypoint Exchange.
-. Search for the "objectstore connector", choose the version of the connector for Mule 3 or Mule 4, and click Install.
-. Follow the prompts to install the connector.
+Install Object Store Connector to add it to a Mule project:
 
-== Use Object Store Connector in an Application
+. In Studio, create a Mule project.
+. In the Mule Palette view, click *(X) Search in Exchange*.
```
Comment (kbostrom) [inline modules/ROOT/pages/osv2-guide.adoc]:
```suggestion
. In the *Mule Palette* view, click *(X) Search in Exchange*.
```

File: modules/ROOT/pages/osv2-guide.adoc [hunk 5]
```diff
@@ -3,141 +3,120 @@ ifndef::env-site,env-github[]
...
 
-*Note:* The information that follows applies only to Mule 3.
+== Use Object Store Connector in an Application
 
-. In Anypoint Studio, click *File > New > Mule Project*.
-. Create a simple flow such as the following:
+. In Studio, create a simple flow such as the following:
 +
+--
+.The screenshot shows (*1*) the *Listener* and *Store* operations in the *os-Simple-Flow* flow.
```
Comment (kbostrom) [inline modules/ROOT/pages/osv2-guide.adoc]: Do we need 1 if there is only one?

File: modules/ROOT/pages/osv2-guide.adoc [hunk 6]
```diff
@@ -3,141 +3,120 @@ ifndef::env-site,env-github[]
...
+. In the details pane, copy the *App url* address.
 
 == Send Data to an Object Store
 
-From a command line prompt, send JSON data to your Mule application using a curl command or similar.
-If you prefer, you can use Postman, a browser extension or other tool instead of the command line.
-For Windows, you can use Postman, curl, or cygwin.
-
-For example:
+From a command line prompt, send JSON data to your Mule application using a curl command:
```
Comment (kbostrom) [inline modules/ROOT/pages/osv2-guide.adoc]:
```suggestion
From a command line prompt, send JSON data to your Mule application using a `curl` command:
```

File: modules/ROOT/pages/osv2-guide.adoc [hunk 7]
```diff
@@ -3,141 +3,120 @@ ifndef::env-site,env-github[]
...
-
+--
+. Save the application.
+. Right-click the application name in *Package Explorer* and select *Anypoint Platform > Deploy to CloudHub*.
+. If prompted, sign in to Anypoint Platform.
+. Click *Deploy Application* and click *Open in Browser*.
+. In Runtime Manager, click the *Status* column for your application.
+. In the details pane, copy the *App url* address.
 
 == Send Data to an Object Store
```
Comment (kbostrom) [inline modules/ROOT/pages/osv2-guide.adoc:78]: "an object store" or "Object Store"?
GLOBALLY search for and evaluate: generic or branded product?

File: modules/ROOT/pages/osv2-guide.adoc [hunk 8]
```diff
@@ -3,141 +3,120 @@ ifndef::env-site,env-github[]
...
-curl -X POST -H "Content-Type: application/json" -d '{ "key": "TestKey01", "value": "This is an object store test" }' "http://application_name.cloudhub.io/store"
+curl -X POST -H "Content-Type: application/json" -d '{ "key": "TestKey01", "value": "Testing" }' "http://app-url/store"
 ----
 
-Change `application_name` to the name you chose when you deployed your application. Each time you submit this command, change the key's name so that each key is unique.
+Change `app-url` to the name you copied from *App url* in Runtime Manager. 
 
 == View Data in an Object Store
 
 To view data in the Object Store:
```
Comment (kbostrom) [inline modules/ROOT/pages/osv2-guide.adoc]: ditto

File: modules/ROOT/pages/osv2-guide.adoc [hunk 9]
```diff
@@ -3,141 +3,120 @@ ifndef::env-site,env-github[]
...
+. Click *Object Store* in the left navigation pane.
+. Click the object store, the partition, and then the key:
++
+.The screenshot shows (*1*) the object store, (*2*) the key, and (*3*) the value in the object store.
 +
-image::osv2-in-nav-bar.png[]
+image::os-rtm-key-in-os.png[Object Store option, key, and value in Runtime Manager]
 +
-The Object Store user interface appears as follows:
+The columns show the Object Store name, partition name, key, and key data.
```
Comment (kbostrom) [inline modules/ROOT/pages/osv2-guide.adoc]: lc "object store"?

File: modules/ROOT/pages/osv2-guide.adoc [hunk 10]
```diff
@@ -3,141 +3,120 @@ ifndef::env-site,env-github[]
...
+The partition groups the object store keys.
 +
-image::osv2-ui.png[]
+The `TestKey01` key that you sent from `curl` appears in the object store with the value of `[binary value] BINARY`.
```
Comment (kbostrom) [inline modules/ROOT/pages/osv2-guide.adoc]:
```suggestion
The `TestKey01` key that you sent from `curl` appears in the object store with the value `[binary value] BINARY`.
```

File: modules/ROOT/pages/osv2-guide.adoc [hunk 11]
```diff
@@ -3,141 +3,120 @@ ifndef::env-site,env-github[]
...
+The value is binary because Mule 4 wraps values in a Mule object, which causes
+them to be visible only as binary in Anypoint Platform.
 +
-* The default Object Store name is `DEFAULT_USER_STORE`.
-* The columns show the Object Store name, partition name, key, and key data. The partition groups object stores keys.
+. To delete a key, hover over a key name, and click the trash can icon.
```
Comment (kbostrom) [inline modules/ROOT/pages/osv2-guide.adoc]:
```suggestion
. To delete a key, hover over a key name and click the trash can icon.
```

File: modules/ROOT/pages/osv2-guide.adoc [hunk 12]
```diff
@@ -3,141 +3,120 @@ ifndef::env-site,env-github[]
...
 +
-. Click the Object Store name, partition name, and key to view its value.
-. To delete a key, hover over a key name, and click the trash can icon. Similarly,
-you can delete a partition or the store itself by hovering and clicking the trash
-can icon.
+Similarly, you can delete a partition or the store itself by hovering and clicking the trash can icon.
```
Comment (kbostrom) [inline modules/ROOT/pages/osv2-guide.adoc]:
```suggestion
Similarly, you can delete a partition or the store itself by hovering over and clicking the trash can icon.
```

File: modules/ROOT/pages/osv2-guide.adoc [hunk 13]
```diff
@@ -3,141 +3,120 @@ ifndef::env-site,env-github[]
...
 
 [NOTE]
 Object Store v2 provides persistent storage of objects with a time to live (TTL) of 30 days.
 
+The Tutorial provides more detailed information about creating object stores in Mule 3 and Mule 4 applications.
```
Comment (kbostrom) [inline modules/ROOT/pages/osv2-guide.adoc]:
```suggestion
The tutorial provides more detailed information about creating object stores in Mule 3 and Mule 4 applications. 
```
or give precise title

**Review by kbostrom — DISMISSED** [review]
Edited

File: modules/ROOT/pages/osv2-guide.adoc (hunk 7 above)
Comment (hannanelson) [reply modules/ROOT/pages/osv2-guide.adoc:78]: an object store (but it's a title).

File: modules/ROOT/pages/osv2-guide.adoc (hunk 8 above)
Comment (hannanelson) [reply modules/ROOT/pages/osv2-guide.adoc]: changed "the" to "an".

**Review by kbostrom — APPROVED** [review]
reapproval

**Pull request merged**